.github/workflows/CodeCov.yml

```diff
@@ -19,7 +19,7 @@ jobs:
     - name: Set up Julia
       uses: julia-actions/setup-julia@latest
       with:
-        version: 1.7.0
+        version: 1.8
```

Comment:
@charleskawczynski Should this be more specific, e.g. 1.8.5?

Reply:
1.8 should be fine